src/zenml/zen_server/deploy/docker/docker_zen_server.py

```diff
@@ -190,6 +191,8 @@ def _get_container_cmd(self) -> Tuple[List[str], Dict[str, str]]:
             SERVICE_CONTAINER_GLOBAL_CONFIG_PATH,
             LOCAL_STORES_DIRECTORY_NAME,
         )
+        env[ENV_ZENML_DISABLE_DATABASE_MIGRATION] = "True"
```

Comment:
Can you explain why this is necessary? To me this seems the same as the local case (and should also be running a docker container which contains the same ZenML version as the local python environment) and therefore the migration is necessary? Or will the migration always happen locally (e.g. triggered by the call to `zenml up --docker` and this is included to save the check if migration should be done?

Reply:
The migration will always happen locally. This is to prevent the server from accidentally migrating a local database and leading to unexpected results (especially useful for us in development).